libs/zlib/gzguts.h

```diff
@@ -19,6 +19,7 @@
 #endif
 
 #include <stdio.h>
+#include <unistd.h>
```

Comment:
You said we can do without zlib so this is not needed, right?

Reply:
PNaCl still needs that file, but we are lucky that PNaCl doesn't require this change, so yes we can probably skip this commit, this whole folder will be removed once we drop PNaCl.